astropy/units/physical.py

```diff
@@ -1,133 +1,530 @@
-# -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-"""
-Defines physical unit names.
-
-This module is not intended for use by user code directly.  Instead,
-the physical unit name of a `Unit` can be obtained using its `ptype`
-property.
-"""
+"""Defines the physical types that correspond to different units."""
 
+import numbers
 
 from . import core
 from . import si
 from . import astrophys
 from . import cgs
-from . import imperial
```

Comment:
Looking at #11975 , it is unclear to me now why this line was removed...

Reply:
Oh my.  This was a huge mistake on my part. I apologize for that. I have no idea how or why I let that slip in there....

Reply:
Probably a tool warned "unused import"?